gromacs/config.py

```diff
@@ -639,11 +639,15 @@ def set_gmxrc_environment(gmxrc):
     then only a warning will be logged. Thus, it should be safe to just call
     this function.
     """
-    envvars = ['GMXPREFIX', 'GMXBIN', 'GMXLDLIB', 'GMXMAN', 'GMXDATA',
-               'GROMACS_DIR', 'LD_LIBRARY_PATH', 'MANPATH', 'PKG_CONFIG_PATH',
-               'PATH']
+    # only v5: 'GMXPREFIX', 'GROMACS_DIR'
+    envvars = ['GMXBIN', 'GMXLDLIB', 'GMXMAN', 'GMXDATA',
+               'LD_LIBRARY_PATH', 'MANPATH', 'PKG_CONFIG_PATH',
+               'PATH',
+               'GMXPREFIX', 'GROMACS_DIR']
+    # in order to keep empty values, add ___ sentinels around result
+    # (will be removed later)
     cmdargs = ['bash', '-c', ". {0} && echo {1}".format(gmxrc,
-               ' '.join(['${0}'.format(v) for v in envvars]))]
+               ' '.join(['___${{{0}}}___'.format(v) for v in envvars]))]
```

Comment:
Btw, it might be better to put the values on new lines and then split on those:
```python
cmdargs = ['bash', '-c', ". {0} && echo {1}".format(gmxrc, 
           '\n'.join(['${{{0}}}'.format(v) for v in envvars]))]
...
    out = out.split('\n')
    for key, value in zip(envvars, out):
             value = value.strip()
             os.environ[key] = value
```

This will work even for subsequent empty lines:
```python
>>> out = "\n".join(["line 1", "line 2", "", "line 4", "", "", "line_5", ""])
>>> out.split("\n")
['line 1', 'line 2', '', 'line 4', '', '', 'line_5', '']
```